hack/scripts/cluster-kernel-upgrade.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Should be used for manual validation. Does not check if correct kernel is installed or if installation failed.
+
+# DUMMY_CNI     - Flag to provide a temporary CNI
+if [ $DUMMY_CNI = "true" ]; then
+    echo "-- Install dummy CNI for nodes to be marked as Ready --"
+    kubectl get pods -Aowide
+    kubectl apply -f https://raw.githubusercontent.com/Azure/azure-container-networking/v1.5.3/hack/manifests/cni-installer-v1.yaml
```

Comment:
The version here is different from the version for privileged-daemonset. Going forward are these going to be same or we need to change them as we release new versions ? If this for testing maybe we can add version parameter to script.